src/base/preferences.cpp

```diff
@@ -1459,10 +1464,9 @@ void Preferences::upgrade()
     QStringList labels = value("TransferListFilters/customLabels").toStringList();
     if (!labels.isEmpty()) {
         QVariantMap categories = value("BitTorrent/Session/Categories").toMap();
-        foreach (const QString &label, labels) {
```

Comment:
There was no violation of the coding style, so this is a purely subjective change...

Reply:
I mean we shouldn't accept such changes. Otherwise, everyone will do some unrelated changes, just formatting some code snippets in its own way.

Reply:
OK. Reverted.